doc/source/whatsnew/v1.1.0.rst

```diff
@@ -98,6 +98,7 @@ Performance improvements
 Bug fixes
 ~~~~~~~~~
 
+- Fixed bug, where the Series representation of an object Index with bools and NaNs was wrong. (:issue:`32146`)
```

Comment:
can move to 1.0.2 (put in regressions section), use double backticks around ``Series``, ``Index`` and ``NaN`` s